lmdeploy/turbomind/chat.py

```diff
@@ -59,16 +72,15 @@ def main(model_path: str,
     if chat_template_cfg is None:
         chat_template_cfg = ChatTemplateConfig(model_name=model_name,
                                                capability=cap)
-        new_kwargs = {}
-        for k, v in kwargs.items():
-            if hasattr(chat_template_cfg, k):
-                setattr(chat_template_cfg, k, v)
-            else:
-                new_kwargs[k] = v
-        kwargs = new_kwargs
+        kwargs = fill_config(chat_template_cfg, kwargs)
+    if engine_cfg is None:
+        engine_cfg = TurbomindEngineConfig(model_name=model_name)
+        kwargs = fill_config(engine_cfg, kwargs)
```

Comment:
Seems not a perfect way to support the feature. Filling arguments to  chat_template is for backward compatibility. Why not just use `**kwargs`?

Reply:
I just fixed it. Thanks.